src/chrome/content/rules/Private-Layer.com.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="Private Layer" platform="cacert" default_off="missing certificate chain">
```

Comment:
my mistake, it's not cacert. the website comes with intermediate cert "GeoTrust DV SSL CA - G4" which causes the cert issue. anyway, I'll re-open this PR when cert issue has been resolved.